internal/constants/constants.go

```diff
@@ -53,3 +54,51 @@ const (
 	// UUIDSize is the length of a UUID, in bytes.
 	UUIDSize = 16
 )
+
+// domain migration flags
+const (
```

Comment:
These can't live in `internal/constants/` because they need to be specified by the caller to `l.Migrate`.